CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - `elasticsearch-discovery` service is not a headless service anymore
```

Comment:
In your commit message:

> Static files were collected only when a custom theme was configured, leading, the `nginx` BC to fail since the  `/edx/app/edxapp/staticfiles` directory does not exist. Static files should always be collected whether a theme has been configured or not.